### ISSUE TITLE
Create new teaser card pattern

### ISSUE THREAD

**Issue by cehfisher** [cehfisher/a11y-style-guide#193]
Noticed missing a teaser image with body text and link pattern that most sites seem to have. Most sites have redundant links, which is a WCAG warning...but also just annoying to users of screen readers.

<img width="962" alt="pattern" src="https://user-images.githubusercontent.com/15240149/34644847-2a9e18d4-f304-11e7-8fe5-0fd27a393c44.png">
